pkg/testing/ess/deployment.go

```diff
@@ -324,23 +399,15 @@ func overallStatus(statuses ...DeploymentStatus) DeploymentStatus {
 //go:embed create_deployment_request.tmpl.json
 var createDeploymentRequestTemplate string
 
-//go:embed create_deployment_csp_configuration.yaml
+//go:embed deployment_csp_configuration.yaml
```

Comment:
I am pretty sure that this file being renamed breaks the following:
* https://github.com/elastic/elastic-agent/blob/73737c9a3c70d9bdbeade078097dcfae1bb61719/.ci/updatecli/updatecli-bump-stack-version.yml#L29
* https://github.com/elastic/elastic-agent/blob/73737c9a3c70d9bdbeade078097dcfae1bb61719/.mergify.yml#L39
* https://github.com/elastic/elastic-agent/blob/73737c9a3c70d9bdbeade078097dcfae1bb61719/test_infra/ess/readme.md#L9

Reply:
Good catch, @pkoutsovasilis – thanks! Will make sure to update all references.